tests/test_pdf_transform.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import json
+from transform.views.test_views import test_message
+from transformers import PDFTransformer
+
+
+class TestPDFTransformer(unittest.TestCase):
+    def test_localised_time(self):
+        with open("./surveys/023.0203.json") as survey:
+            response = json.loads(test_message)
+            pdf_transformer = PDFTransformer(survey, response)
+
+            expected_date = "Sat 12 Mar 10:39:40am 2016"
+            actual_date = pdf_transformer.get_localised_date()
+
+            self.assertEqual(expected_date, actual_date)
+
+            expected_date = "Sat 12 Mar 02:39:40am 2016"
+            actual_date = pdf_transformer.get_localised_date(timezone='US/Pacific')
+
+            self.assertEqual(expected_date, actual_date)
+
+            expected_date = "Sat 12 Mar 18:39:40pm 2016"
```

Comment:
My point was how would somebody who reviews the image know what the time 10:39:40 represents?